docs/adr/0007-dependent-deletion.md

```diff
@@ -0,0 +1,90 @@
+# 7. Dependent Deletion
```

Comment:
Yeah, sorry. I'm going to prefer time of merge, just because it doesn't make sense to have a missing number in there if one of the bigger ADRs takes a long time.

Reply:
Sure - then these probably should not be assigned a number until after approval to indicate that is pending.

Reply:
Fair enough - renamed :-).